src/pages/registration/references_tab/components/SearchContainerField.tsx

```diff
@@ -34,70 +35,116 @@ const SearchContainerField = (props: SearchContainerFieldProps) => {
+                setFieldValue(field.name, `${API_URL}${getRegistrationPath(inputValue.id)}`);
+              } else {
+                setSearchTerm('');
+                setFieldValue(field.name, '');
+              }
+            }}
+            loading={isLoadingSearchContainerOptions || isLoadingSelectedContainer}
+            getOptionLabel={(option) => option.title}
+            renderOption={(option, state) => (
+              <StyledFlexColumn>
+                <Typography variant="subtitle1">
+                  <EmphasizeSubstring text={option.title} emphasized={state.inputValue} />
+                </Typography>
+                <Typography variant="body2" color="textSecondary">
+                  <Truncate lines={1}>
+                    {option.publicationDate.year && displayDate(option.publicationDate)}
+                    {option.publicationDate.year && option.contributors.length > 0 && ' - '}
+                    {option.contributors.map((contributor) => contributor.name).join('; ')}
+                  </Truncate>
+                </Typography>
+              </StyledFlexColumn>
+            )}
+            renderInput={(params) => (
+              <AutocompleteTextField
+                {...params}
+                label={props.label}
+                required
+                isLoading={isLoadingSearchContainerOptions || isLoadingSelectedContainer}
+                placeholder={props.placeholder}
+                dataTestId="container-search-field"
+                showSearchIcon
+                errorMessage={meta.touched && !!meta.error ? meta.error : undefined}
+              />
+            )}
+          />
+        )}
+      </Field>
+      {selectedContainer?.reference?.publicationContext && (
+        <SelectedContainerSummary publicationContext={selectedContainer.reference.publicationContext} />
       )}
-    </Field>
+    </>
+  );
+};
+
+interface SelectedContainerSummaryProps {
+  publicationContext: SearchPublicationContext;
+}
+
+const SelectedContainerSummary = ({ publicationContext }: SelectedContainerSummaryProps) => {
+  const { t } = useTranslation('registration');
+  const { publisher, title, url, onlineIssn, printIssn, level } = publicationContext;
+  const levelValue = level ? levelMap[level] : null;
+
+  return (
+    <>
+      {title && (
+        <Typography color="primary">
+          {t('references.journal')}: {title}
+        </Typography>
+      )}
+      {publisher && (
+        <Typography color="primary">
+          {t('common:publisher')}: {publisher}
+        </Typography>
+      )}
+      {url && (
+        <Typography color="primary" component="a" href={url}>
+          {url}
+        </Typography>
+      )}
+      {levelValue && (
+        <Typography color="primary">
+          {t('references.level')}: {levelValue}
+        </Typography>
+      )}
+      {(printIssn || onlineIssn) && (
+        <Typography color="primary">
```

Comment:
jeg liker egentlig ikke at vi setter primary til Typography, siden primary er blå/hvit.
ville likt bedre om vi hadde satt `color=textPrimary` som default prop for Typography i theme, og oppdatert text-verdier for dark theme slik at textPrimary blir hvit når theme er dark

Reply:
tenkte på det jeg også - kan godt endre til å overskrive textPrimary isteden. En del andre komponenter må også overskrives da (select, checkbox, radiobutton osv). men det er kanskje verdt det uansett for at det skal bli mer "riktig"

Reply:
sant, radio osv må settes separat igjen da ja...:/ men jeg syns uansett det gir mer mening